backend/benefit/companies/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin  # noqa
+
+# Register your models here.
```

Comment:
maybe register model / add basic ModelAdmin?

Reply:
Yeah, totally didn't look into this file :D Fixed now